src/api/generate-xlsx/map-application-to-XLSX/map-export-contract/map-final-destination/index.test.ts

```diff
@@ -33,7 +33,7 @@ describe('api/generate-xlsx/map-application-to-xlsx/map-export-contract/map-fina
 
       const expected = [
         xlsxRow(String(FIELDS.EXPORT_CONTRACT[FINAL_DESTINATION_KNOWN]), mapYesNoField({ answer: finalDestinationKnownAnswer })),
-        xlsxRow(String(CONTENT_STRINGS[FINAL_DESTINATION]), String(country)),
+        xlsxRow(String(CONTENT_STRINGS[FINAL_DESTINATION].SUMMARY?.TITLE), String(country.name)),
```

Comment:
do we need the ?. since its in the CONTENT_STRINGS and not an optional application field?

Reply:
Yes, we have this throughout. the dynamic field ID reference could be undefined.

Only way to remove this is to introduce typings for all `CONTENT_STRINGS`.